src/core/ServiceAccount.ts

```diff
@@ -77,8 +78,8 @@ export class ServiceAccount {
   /**
    * Fetch all subscriptions
    */
-  public async getSubscriptions(): Promise<Subscription[]> {
-    const subscriptions: Subscription[] = await Subscription.get(this.requestBuilder);
+  public async getSubscriptions(events: EventName[]): Promise<Subscription[]> {
```

Comment:
It should be optional, otherwise it is a breaking change

Reply:
According to me it is a required parameter. What do tou think ?

Reply:
Not necessarily, a user can get subscriptions without filtering them 🙂

Reply:
Yes I see your point. But is it normal that a connector can see/change other connectors subscriptions ?

Reply:
No, but that's another subject 😉